### DISCUSSION TITLE
Web3Modal Lab

### DISCUSSION THREAD

**Discussion by communityvolunteer** [WalletConnect/walletconnect-monorepo#2398] [Developer Support]
Hey all. Quick question, when I go to https://web3modal-dev.pages.dev/ on mobile (android) and click Connect Wallet, then choose any of my wallets how come it opens the wallet-app (metamask, trust, coinbase etc) but then nothing else happens lol? 

Sorry if that's a newb question..

**Comment by finessevanes** [comment]
All questions are always welcomed 😄 What playground are you selecting?
